test/library/packages/UnitTest/AssertGreater/AssertGreaterTest.good

```diff
@@ -23,7 +23,7 @@ Error Caught in Array Others Diff Shape High Dim
 AssertionError: assert failed - First element is of shape (5, 7) and Second is of shape (7, 5)
 ================================================================================
 Error Caught in Array Others Diff Shape 1D
-AssertionError: assert failed - First element is of shape (5) and Second is of shape (3)
+AssertionError: assert failed - First element is of shape (5,) and Second is of shape (3,)
```

Comment:
I think this change is an improvement to be more consistent with how tuples are actually printed. It seems that ``ChapelIOStringifyHelper`` has its own stringify implementations for certain well-known types, and it wasn't updated when we started writing 1-tuples with a trailing comma.